tests/test_project_files.py

```diff
@@ -316,8 +319,12 @@ def test_dragon_drop(self, driver, default_project, session, provider, modifier_
 
             # Wait for 5 seconds for Copying message to show
             WebDriverWait(driver, 5).until(EC.visibility_of_element_located((By.CLASS_NAME, 'text-muted')))
-            # Wait a maximum of 20 seconds for Copying message to resolve
-            WebDriverWait(driver, 20).until(EC.invisibility_of_element_located((By.CLASS_NAME, 'text-muted')))
+
+            try:
+                # Wait a maximum of 30 seconds for Copying message to resolve
+                WebDriverWait(driver, 30).until(EC.invisibility_of_element_located((By.CLASS_NAME, 'text-muted')))
+            except TimeoutException:
+                files_page.goto()
```

Comment:
There is another `files_page.goto()` right after this `try ... except ...` block. Maybe a `pass` here when exception happens?

Reply:
Implemented. Great suggestion!